configure.ac

```
@@ -961,14 +961,16 @@ case "$enable_delayacct" in
          enable_delayacct=no
       else
          old_CFLAGS="$CFLAGS"
-         CFLAGS="$CFLAGS -I/usr/include/libnl3"
+         htop_config_cflags=`$PKG_CONFIG --cflags libnl-3.0 2>/dev/null` || continue
```

AFAICS, this should be `|| true`, as there is no loop.
```suggestion
         htop_config_cflags=`$PKG_CONFIG --cflags libnl-3.0 2>/dev/null` || true
```